tests/agent_test.py

```diff
@@ -0,0 +1,72 @@
+import logging
+from typing import Any, Dict, List
+
+import pytest
+import yaml
+from fixieai import FixieClient
+
+
+@pytest.fixture
+def client():
```

Comment:
how does the client authenticate?

Reply:
Using the FIXIE_API_TOKEN env var. This is already what is used when deploying the agents in the GitHub Actions workflow, so when I add the `pytest` run it will do the same.

Reply:
right gotcha